podman.mk

```diff
@@ -0,0 +1,23 @@
+################################################################################
+#
+# podman
+#
+################################################################################
+
+PODMAN_VERSION = 5.0.3
```

Comment:
This sort of thing is not done. A version string in any project is defined exactly once, in only one place, and never duplicated. Anything that references that version string shall obtain it from the One True Source. In our repo, the convention is something along these lines:
```bash
grep '^const RawVersion' version/rawversion/version.go | cut -d\" -f2
```

Reply:
Alternative: Compile and run `test/version//version`, it will output the version number w/o any need to maintain grep's interpretation of golang.  Though it's unlikely to change, so grep is probably fine for now.